handlers/feedback_test.go

```diff
@@ -240,7 +194,7 @@ func Test_addFeedback(t *testing.T) {
 				},
 			}}
 		Convey("When addFeedback is called", func() {
-			addFeedback(w, req, isPositive, mockRenderer, mockSender, from, to, lang, mockNagivationCache)
+			addFeedback(w, req, mockRenderer, mockSender, from, to, lang, mockNagivationCache)
 			Convey("Then the renderer is not called", func() {
 				So(len(mockRenderer.BuildPageCalls()), ShouldEqual, 0)
 			})
```

Comment:
Two lines below Convey description should be `Then the email sender is not called`